resources/cdk/go-apprunner/README.md

```diff
@@ -0,0 +1,97 @@
+# Golang: AppRunner with ECR via CDK
+
+This is an example of building Golang applications on top of AppRunner.
+
+AppRunner does not natively support Go, however it is possible to automate the deployment of a Go application using the
+Elastic Container Registry. Here, a simple application written in Go, is packaged into a Docker container.
+
+* The docker container is built in two passes: The first builds the Golang application for the target linux-amd64 environment.
+  The second stage assembles the final container used by the application: A barebones Go application and supporting environment.
+* This application is built using [fiber](://gofiber.io/) however any HTTP framework will do. 
+* AppRunner automatically procures HTTPS certificates. 
+
+
+## :important: Important
+
+* While this sample has been tested in multiple regions, you will need to verify that AppRunner and the Elastic Container Registry are
```

Comment:
If you are talking about a region in AWS, use "AWS Region" on first use, "Region" thereafter.